<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomKernelView</class>
 <widget class="QDialog" name="CustomKernelView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PhotoLaboratory Custom Kernel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/PhotoLaboratoryIcon.ico</normaloff>:/PhotoLaboratoryIcon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color:  #18122B;
}

QDoubleSpinBox {
	background-color:#18122B;
	color: white;
	border: 1px solid gray
}

QPushButton {
	background-color: #18122B;
	color: white;
	border: 1px solid gray;
	border-radius: 10px;
}

QPushButton::hover {
	background-color: #393053;
}


QToolTip {
	color: white;
	background-color: #393053;
	border: 0px;
}

QTableView {
	background-color:#393053;
}

QTableView::item {
	background-color: #393053;
	color: white;
}

QHeaderView::section {
	background-color: #393053;
	color: white;
}

QTableView QTableCornerButton::section {
	background-color: #18122B;
	color: white;
}

QScrollBar::vertical {
	border: none;
	background-color: #18122B;
	width: 10px;
	margin: 25px 0 0px 0;
}

QScrollBar::handle:vertical {
	border-radius: 4px;
	border-color: rgba(62, 61, 105,75%);
    border-width: 1px;
	border-style: solid;
	background-color: rgba(62, 61, 105,75%);
    min-height: 25px;
}

QScrollBar::add-line:vertical {
	width: 0px;
	height: 0px;	
}

QScrollBar::sub-line:vertical {
	width: 0px;
	height: 0px;	
}

QScrollBar::add-page:vertical {
	background-color: transparent;
}

QScrollBar::sub-page:vertical {
	background-color:  transparent;
}

QScrollBar::horizontal {
	border: none;
	background-color: #18122B;
	width: 10px;
	margin: 25px 0 0px 0;
}

QScrollBar::handle:horizontal {
	border-radius: 4px;
	border-color: rgba(62, 61, 105,75%);
    border-width: 1px;
	border-style: solid;
	background-color: rgba(62, 61, 105,75%);
    min-height: 25px;
}

QScrollBar::add-line:horizontal {
	width: 0px;
	height: 0px;	
}

QScrollBar::sub-line:horizontal {
	width: 0px;
	height: 0px;	
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="6">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinCols">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>16.000000000000000</double>
     </property>
     <property name="value">
      <double>3.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxRows">
     <property name="styleSheet">
      <string notr="true">

</string>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>16.000000000000000</double>
     </property>
     <property name="value">
      <double>3.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QPushButton" name="pushButtonReset">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QPushButton" name="pushButtonOK">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
